frontend/app/routes/_public+/intake+/$id+/demographics-part2.tsx

```diff
@@ -67,6 +69,35 @@ export default function DemographicsPart2() {
           </div>
         )}
 
+        {lastTimeDentistVisitTypes.length > 0 && (
+          <div className="my-6">
+            <InputRadios
+              id="mouth-pain-type"
+              name="mouthPainType"
+              legend={t('demographics-oral-health-questions:part2.question8')}
+              options={lastTimeDentistVisitTypes.map((lastTimeDentistVisitType) => ({
+                children: getNameByLanguage(i18n.language, lastTimeDentistVisitType),
+                value: lastTimeDentistVisitType.id,
+              }))}
+              required
+            />
+          </div>
+        )}
+
+        {avoidedDentalCostTypes.length > 0 && (
+          <div className="my-6">
+            <InputRadios
+              id="mouth-pain-type"
+              name="mouthPainType"
+              legend={t('demographics-oral-health-questions:part2.question9')}
```

Comment:
**Nitpick:** This was also mentioned in the Slack channel, but consider (maybe in a future PR?) renaming the key to reflect the subject of the text rather than the question number. 

It's possible down the line that Question 9 can be switched with Question 13, and developers will leave the key name as `question9`, causing confusion.